front/app/components/UI/ImagesDropzone/index.tsx

```diff
@@ -213,11 +212,7 @@ interface InputProps {
   borderRadius?: string;
   removeIconAriaTitle?: string;
   className?: string;
-}
-
-interface Props extends InputProps {
-  maxNumberOfImages: number;
-  maxImageFileSize: number;
+  previewOverlayElement?: JSX.Element | null;
```

Comment:
Not super happy about this way for adding an overlay... Could be part of the component, but short on time.